pkg/audit/manager.go

```diff
@@ -95,6 +95,16 @@ func New(ctx context.Context, mgr manager.Manager, opa *opa.Client) (*Manager, e
 		ctx:      ctx,
 		reporter: reporter,
 	}
+
+	if *auditInterval < 0 {
```

Comment:
Generally invalid arguments -> fail with error + documentation

Reply:
fail the program with error is what I meant, sorry for the ambiguity. It looks like there is no easy way of failing-on-parse short of writing your own parser. Does switching to Uint lead to fail-with-usage info a user puts a negative value?